app/staging/domain/offender-manager.js

```diff
@@ -0,0 +1,12 @@
+class WorkloadOwner {
+  constructor (id, omKey, omForename, omSurname, omTypeId, omGradeCode) {
+    this.id = id
+    this.omKey = omKey
+    this.omForename = omForename
+    this.omSurname = omSurname
+    this.omTypeId = omTypeId
+    this.omGradeCode = omGradeCode
```

Comment:
Do we know any more about OM grades? I'm wondering if this would be best represented as an ID pointing to reference data?